src/Sarif.Driver/Sdk/SdkResources.resx

```diff
@@ -120,48 +120,32 @@
   <data name="MSG1001_AnalyzingTarget_Description" xml:space="preserve">
     <value>An analysis target was analyzed.</value>
   </data>
-  <data name="ERR0998_ExceptionInAnalyze" xml:space="preserve">
-    <value>An exception was raised analyzing '{0}' for check '{1}' (which has been disabled for the remainder of the analysis). The exception may have resulted from a problem related to parsing image metadata and not specific to the rule, however. Exception information:
-{2}</value>
+  <data name="Notification_ExceptionInAnalyze" xml:space="preserve">
+    <value>An exception was raised analyzing '{0}' for check '{1}' (which has been disabled for the remainder of the analysis). The exception may have resulted from a problem related to parsing image metadata and not specific to the rule, however.</value>
   </data>
-  <data name="ERR0998_ExceptionInCanAnalyze" xml:space="preserve">
-    <value>An exception was raised attempting to determine whether '{0}' is a valid analysis target for check '{1}' (which has been disabled for the remainder of the analysis). The exception may have resulted from a problem related to parsing the analysis target and not specific to the rule, however. Exception information:
-{2}</value>
+  <data name="Notification_ExceptionInCanAnalyze" xml:space="preserve">
+    <value>An exception was raised attempting to determine whether '{0}' is a valid analysis target for check '{1}' (which has been disabled for the remainder of the analysis). The exception may have resulted from a problem related to parsing the analysis target and not specific to the rule, however.</value>
   </data>
-  <data name="ERR0997_ExceptionCreatingLogFile" xml:space="preserve">
-    <value>An exception was raised attempting to create output file: '{0}'. Exception information:
-{1}</value>
+  <data name="Notification_ExceptionCreatingLogFile" xml:space="preserve">
+    <value>Could not create output file '{0}'.</value>
   </data>
-  <data name="ERR0999_UnhandledEngineException" xml:space="preserve">
-    <value>An exception was raised during analysis:
-{0}</value>
+  <data name="Notification_UnhandledEngineException" xml:space="preserve">
+    <value>An exception was raised during analysis.</value>
   </data>
-  <data name="ERR0999_AnalysisHalted_Description" xml:space="preserve">
-    <value>Analysis halted and was incomplete due to an unrecoverable error such as an unhandled exception.</value>
+  <data name="Notification_ExceptionInInitialize" xml:space="preserve">
+    <value>An exception was raised initializing check '{0}' (which has been disabled for the remainder of the analysis).</value>
   </data>
```

Comment:
The "*_Description" resource strings were for the Rule descriptions in Errors.cs. But Errors.cs does use Rules any more.